Makefile

```diff
@@ -108,3 +111,8 @@ create-tmp-env-ci:
 # 16 is exit code for critical https://classic.yarnpkg.com/lang/en/docs/cli/audit
 audit:
 	bash -c 'yarn audit --level critical; [[ $$? -ge 16 ]] && exit 1 || exit 0'
+
+mine:
```

Comment:
when reading mine, I thought "this is mine" and was confusing about the action - versus "mining". maybe `mine-blocks` could be more explicit?